### DISCUSSION TITLE
How to add the github logo link on navbar?

### DISCUSSION THREAD

**Discussion by BattlefieldDuck** [dotnet/docfx#9644] [Q&A] [answered]
![Screenshot 2024-01-23 192209](https://github.com/dotnet/docfx/assets/29337428/5a6370bd-3fef-4422-9c01-edd7be387962)

thanks.

**Comment by paulushub** [comment, marked as answer]
Create a `main.js` in say template/public. See this https://github.com/dotnet/docfx/tree/main/docs/template/public
Add the template to the `docfx.json` file: See https://github.com/dotnet/docfx/blob/main/docs/docfx.json

**Reply by BattlefieldDuck** [reply]
Thank you so much!

![image](https://github.com/dotnet/docfx/assets/29337428/7523f7c5-4f89-40d4-abfe-3179852ceef3)
